eng/Packages.Data.props

```diff
@@ -99,9 +99,9 @@
     <PackageReference Update="System.Threading.Channels" Version="6.0.0" />
     <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Update="System.Text.Json" Version="6.0.11" />
-    <PackageReference Update="System.Text.Encodings.Web" Version="6.0.1" />
+    <PackageReference Update="System.Text.Encodings.Web" Version="8.0.0" />
```

Comment:
I'm willing to bet that we can strip this out entirely.   I think it was only included in any project as a dependency because of a CVE that required we pull up the version used by STJ.    Can we test removing it, either here or as a later part of the process?

Reply:
Ah good point. Let me try

Reply:
I'm going to open a separate PR for simplifying dependency experimentation